code/__DEFINES/mobs.dm

```diff
@@ -945,9 +947,10 @@ GLOBAL_LIST_INIT(layers_to_offset, list(
 #define HEAL_AFFLICTIONS (HEAL_NEGATIVE_DISEASES|HEAL_NEGATIVE_MUTATIONS|HEAL_ALL_REAGENTS|HEAL_STATUS|HEAL_CC_STATUS)
 
 /// Full heal that isn't admin forced
-#define HEAL_ALL ~(HEAL_ADMIN|HEAL_RESTRAINTS)
+#define HEAL_ALL ~(HEAL_ADMIN|HEAL_RESTRAINTS|HEAL_REPLACE_ORGANS)
 /// Heals everything and is as strong as / is an admin heal
-#define ADMIN_HEAL_ALL ALL
+#define ADMIN_HEAL_ALL (ALL & ~(HEAL_REPLACE_ORGANS))
+#define ADMIN_HEAL_ALL_REPLACE_ORGANS ALL
```

Comment:
Just delete the second define

Reply:
`HEAL_REPLACE_ORGANS` probably doesn't even need to exist anyways; if someone wants that effect they can just call `regenerate_organs`

Reply:
`HEAL_REPLACE_ORGANS` probably doesn't even need to exist anyways; if someone wants that effect they can just call `regenerate_organs`

Reply:
Wasn't sure if there are cases we actually need it, but having the go-ahead to kill it is great